WcaOnRails/spec/controllers/delegate_reports_controller_spec.rb

```diff
@@ -81,19 +81,15 @@
       expect(comp.delegate_report.remarks).to eq "My newer remarks"
       expect(comp.delegate_report.posted?).to eq true
       expect(comp.delegate_report.posted_by_user_id).to eq user.id
+      # Check if the discussion_url was set
+      expect(comp.delegate_report.discussion_url).to eq "https://groups.google.com/forum/#!topicsearchin/wca-delegates/" + URI.encode(comp.name)
 
       # Try to update the report when it's posted.
       post :update, competition_id: comp.id, delegate_report: { remarks: "My newerer remarks" }
       comp.reload
       expect(comp.delegate_report.remarks).to eq "My newer remarks"
     end
 
-    it "updating report sets the discussion_url" do
```

Comment:
One idea would be to change this test to say "does set update discussion_url" when updating report.